mantidproject/mantid
# Change SaveMD structure for MDHisto

Ticket [#11362](http://trac.mantidproject.org/mantid/ticket/11362), also goes part of the way to resolving #129

This is version 2 of SaveMD and will change the structure of how a MDHisto will be saved. This will help users with using the output from SaveMD in other software and will make the files more NeXus compliant. This doesn't change how MDEvents are saved.

Two big changes, the data (signal, errors, etc) are now stored as multi dimensional arrays reflecting the actually structure of the data, previously it was stored just a single dimension. And the axis are no longer stored as an XML string attribute to the MDHistoWorkspace group but as an data array as per the nexus standard.

**To Test:**
- Try saving an MDHisto workspace with SaveMD and look at how the data is stored in the nexus file.
- Make sure LoadMD can load data saved with SaveMD and SaveMDv1.
- You should check that SaveMD still works for MDEvents.
